### PR TITLE
Add docker CLI instructions for host address

### PR DESCRIPTION
**Pull request by longquanzheng**
We had that before but somehow lost it when migrating to a different branch: https://github.com/uber/cadence/pull/1869/files

### CODE REVIEW

File: docs/08_cli.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ Example of using the docker image to describe a domain:
 docker run --rm ubercadence/cli:master --domain samples-domain domain describe
 ```
 
+You could use docker image `ubercadence/cli`, by replacing all the following `./cadence ...` with `docker run --rm ubercadence/cli:master --address <HOST_DNS>:7933 ...` .
```
Comment (dmetzgar) [inline docs/08_cli.md]: I think you're not reading this section. Check out original line 24.

Comment (dmetzgar) [reply docs/08_cli.md]: This line is unnecessary and the <HOST_DNS> part is confusing.

File: docs/08_cli.md [hunk 2]
```diff
@@ -14,6 +14,14 @@ Example of using the docker image to describe a domain:
 docker run --rm ubercadence/cli:master --domain samples-domain domain describe
 ```
 
+You could use docker image `ubercadence/cli`, by replacing all the following `./cadence ...` with `docker run --rm ubercadence/cli:master --address <HOST_DNS>:7933 ...` . 
+
+You may get "connection refused" error for 18.03 onwards versions of docker. You have to use host.docker.internal. For more info check https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds
```
Comment (dmetzgar) [inline docs/08_cli.md]: This part makes sense but I would reword it:
> On Docker versions 18.03 and later, you may get a "connection refused" error. You can work around this by setting the host to "host.docker.internal" (see [here](https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds) for more info).

Comment (longquanzheng) [reply docs/08_cli.md]: done
